static/app/views/performance/newTraceDetails/traceGuards.tsx

```diff
@@ -23,6 +23,16 @@ export function isSpanNode(
   );
 }
 
+export function isEAPTransaction(value: TraceTree.NodeValue): value is TraceTree.EAPSpan {
+  return !!(value && 'is_transaction' in value && value.is_transaction);
+}
+
+export function isEAPTransactionNode(
+  node: TraceTreeNode<TraceTree.NodeValue>
+): node is TraceTreeNode<TraceTree.EAPSpan> {
+  return !!(isEAPTransaction(node.value) && node.value.is_transaction);
```

Comment:
```suggestion
  return !!(isEAPTransaction(node.value);
```
doesn't `isEAPTransaction` already check if `node.value.is_transaction)` is true?

Reply:
Yep, good catch